src/desktop/db/OfflineDb.ts

```diff
@@ -137,7 +137,7 @@ export class OfflineDb {
 	getIdsInRange(type: string, listId: Id): Array<Id> {
 		const range = this.getRange(type, listId)
 		if (range == null) {
-			throw new Error(`no range exists for ${type} and list ${listId}`)
+			return []
```

Comment:
Is there a reason to change this?

Reply:
it was just for some tests but it's not necessary